students/15th/hayoung/js/login.js

```diff
@@ -0,0 +1,24 @@
+const idInput = document.getElementById('id'),
+passwordInput = document.getElementById('password'),
+loginBtn = document.getElementsByClassName('loginBtn'),
+loginForm = document.getElementsByClassName('loginForm');
+
+loginBtn.disabled = true;
+
+let eventStart = () => {
+    loginForm.addEventListener('keyup',checkLoginBtn);
+}
+
+let checkLoginBtn = () => {
+    let isIdOn = idInput.value.length > 0;
+    let isPasswordOn = passwordInput.value.length > 0;
+    let loginBtnOn = () => {
+        loginBtn.disabled = false;
+        loginBtn.style.backgroundColor = '#0095f6';
+    };
+    let result = isIdOn ? (isPasswordOn ? loginBtnOn() : null) : null;
```

Comment:
```checkLoginBtn```의 함수가 따로 return 하는 값이 없다면 result라는 변수를 만들 필요가 없어 보입니다.

그리고 의도하신 바다 ```isIdOn```과 ```isPasswordOn``` 변수가 true일 때 ```loginBtn```의 속성과 style을 바꾸는 것이라면 다음과 같이 할 수 있습니다.

```suggestion
    const isIdOn = idInput.value.length > 0;
    const isPasswordOn = passwordInput.value.length > 0;
    const isValid = isIdOn && isPasswordOn;
    
    loginBtn.disabled = !isValid
    loginBtn.style.backgroundColor = isValid ? "#0095f6" : "이전 배경 색"
```